app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,25 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
+
+    def player_info(self) -> str:
+        return (
+            f"Elf ranger {self.nickname}. "
+            f"{self.nickname} has bow of the "
+            f"{self._bow_level} level"
+        )
+
+    def play_elf_song(self) -> str:
```

Comment:
The `play_elf_song` method is defined to return a `str`, but it currently does not return any value. Consider returning a string or changing the return type to `None`.